builtin/credential/okta/path_users.go

```diff
@@ -0,0 +1,139 @@
+package okta
```

Comment:
Sorry for being late to the party, but I am in favor of adding policies per-user.